src/HotChocolate/Core/src/Types/Types/Directives/CostTypeInterceptor.cs

```diff
@@ -141,7 +141,7 @@ private static bool IsCostDirective(DirectiveDefinition directive)
     /// <summary>
     /// Defines if a resolver is possible fetching data and causing higher impact on the system.
     /// </summary>
-    private static bool IsDataResolver(ObjectFieldDefinition field)
```

Comment:
I think we should move that into a utility ... having this live in the interceptor but being used by other things would be strange. I will have a look at some place where we can put this one.